src/screens/Survey/checkbox.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { Text, View, Button,StyleSheet,TouchableHighlight,TouchableOpacity, Image, BackHandler} from 'react-native';
+import PropTypes from 'prop-types';
+
+class checkbox extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      responseList : ["stronglyDisagree","disagree","noOpinion","agree","stronglyAgree"],
```

Comment:
Wouldn't it be simpler to just save a number from -2 to 2 that corresponds with the response?

Reply:
Yeah, so this part is specifically for the checkbox. So the way I set it up is by having all those objects as some sort of form of one and off switches. In that way, when you click on a certain option it stays there. I guess I could assign a number to each button and then compare the values? That could be easier.

Reply:
I think it would be good to separate the values we actually store from the labels we put on the checkboxes